content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,416 @@
+|              | `--embed-runtime-image-name=NAME`    |  The image name and tag of a container to include in the package. Only for provider and function packages. |
+|              | `--embed-runtime-image-tarball=PATH` |  The filename of a container to include in the package. Only for provider and function packages.                              |
+| `-e`         | `--examples-root="./examples"`       |  The path to a directory of examples related to the package.                               |
+|              | `--ignore=PATH,...`                  |  List of files and directories to ignore.                              |
+| `-o`         | `--package-file=PATH`                |  Directory and filename of the created package.                             |
+| `-f`         | `--package-root="."`                 |  Directory to search for YAML files.                              |
+{{< /table >}}
+
+The `crossplane xpkg build` command recursively looks in the directory set by 
+`--package-root` and attempts to combine any files ending in `.yml` or `.yaml` 
+into a package.
+
+All YAML files must be valid Kubernetes manifests with `apiVersion`, `kind`, 
+`metadata` and `spec` fields. 
+
+#### Ignore files
+
+Use `--ignore` to provide a list of files and directories to ignore.  
+
+For example,  
+`crossplane xpkg build --ignore="./test/*,kind-config.yaml"`
+
+#### Set the package name
+
+`crossplane` automatically names the new package a combination of the 
+`metadata.name` and a hash of the package contents and saves the contents 
+in the same location as `--package-root`. Define a specific location and 
+filename with `--package-file` or `-o`.  
+
+For example,  
+`crossplane xpkg build -o /home/crossplane/example.xpkg`.
+
+
+#### Include examples
+
+Include YAML files demonstrating how to use the package with `--examples-root`. 
+
+[Upbound Marketplace](https://marketplace.upbound.io/) uses files included with 
+`--examples-root` as documentation for published packages.
+
+#### Include a runtime image
+
+Functions and Providers require YAML files describing their dependencies and 
+settings as well as a container image for the function or provider controller.
+
+Using `--embed-runtime-image-name` downloads a specified container image and 
+includes the image inside the function or provider package. Use the 
+
+The `--embed-runtime-image-tarball` flag includes a local OCI image tarball 
+inside the function or provider package.
+
+
+### xpkg install
+
+Download and install Crossplane packages with  `crossplane xpkg install`.
+Crossplane packages.  
+
+By default the `crossplane xpkg install` command uses the Kubernetes 
+configuration defined in `~/.kube/config`.  
+
+Define a custom Kubernetes configuration file location with the environmental 
+variable `KUBECONFIG`.
+
+Specify the package kind, package file and optionally a name to give the package 
+inside Crossplane.
+
+`crossplane xpkg install <package-kind> <package-file> [<optional-name>`
```

Comment:
I would not call this a package file. It's not a file, and I think this oversimplification will be confusing to anyone familiar with OCI images and registries. I struggled with naming for this too, and landed on just calling it "a package".

Other commands take flags/arguments like `--package-file` and in those cases they accept an actual `.xpkg` file.

"package tag" or "package reference" could also be good here. The former is more accurate (technically you could give a digest, not a tag, like `crossplane-contrib/provider-foo@sha256:...`) but the latter is probably good enough to be meaningful to most people.

Reply:
I also like how this is framed in the provider docs:

> Install a Provider with a Crossplane Provider object setting the spec.package value to **the location of the provider package**.

Reply:
Updated to

`crossplane xpkg install <package-kind> <registry package name and tag> [<optional-name>`